static/app/views/replays/detail/layout/utils.tsx

```diff
@@ -0,0 +1,26 @@
+import theme from 'sentry/utils/theme';
+import {LayoutModes} from 'sentry/views/replays/detail/layout';
+
+const RECOMMENDED_WIDTH = 1700;
+const MIN_RECOMMENDED_WIDTH = 1024;
+const MIN_RECOMMENDED_HEIGHT = 960;
+
+export const getDefaultLayout = (): LayoutModes => {
+  const {innerWidth, innerHeight} = window;
+  const sidebarWidth = parseInt(theme.sidebar.expandedWidth, 10);
+  const windowsWidth = innerWidth - sidebarWidth;
+
+  if (windowsWidth < innerHeight) {
```

Comment:
> If the height is larger than we choose topbar.
> If the width (minus the sidebar) is larger then we choose one of the sidebars.

Done! Also added some new consts to make the magic numbers easier to understand its functionality.